app/controllers/slack_controller.rb

```diff
@@ -25,8 +25,20 @@ def event
     head :ok
   end
 
+  def command
+    list_admins if params[:command] == "admins"
+  end
+
   protected
 
+  def list_admins
+    client = Slack::Web::Client.new
+    admins = []
+    response = client.users_list.members
```

Comment:
I suspect it would be reasonable to cache this response to avoid the (presumably slow) http call most of the time. If it's not noticeably slow to begin with, it's probably not worth bothering yet.

Reply:
rather, the admins list, not the full users list, which changes often

Reply:
I’m on my phone so haven’t looked at the whole thing but no reason to make an API call. SlackUser has scopes to pull human admins. That’s updated hourly and cached in the DB.

Reply:
👍 on both counts, will fix.